pkg/kubelet/apis/config/types.go

```diff
@@ -431,3 +434,11 @@ type SerializedNodeConfigSource struct {
 	// +optional
 	Source v1.NodeConfigSource
 }
+
+// LoggingConfig contains logging options
+type LoggingConfig struct {
+	// LoggingFormat Flag specifies the structure of log messages.
+	// default value of loggingFormat is `text`
+	// Refer [Logs Options](https://github.com/kubernetes/component-base/blob/master/logs/options.go) for more information.
+	LoggingFormat string
```

Comment:
name this `Format`

Reply:
changed it to Format in LoggingConfiguration struct